domains/transfer-agreements/TransferAgreements.API/API/ApiModels/Responses/TransferAgreementsResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections.Generic;
+
+namespace API.ApiModels.Responses
+{
+
+    public record TransferAgreementsResponse(List<TransferAgreementDto> TransferAgreements);
```

Comment:
To make this inline with Certificates-domain, could this property be named `Result` instead of `TransferAgreements`?

Reply:
```suggestion
    public record TransferAgreementsResponse(List<TransferAgreementDto> Result);
```

Reply:
Renamed